src/bugsnag/ErrorPrompt.tsx

```diff
@@ -0,0 +1,123 @@
+// NOTE: do not include any other files here, try to keep this component separate from the rest
```

Comment:
> Given this is imported in the main `index.js` it'll be bundled into all of the other dependencies that are included there as well anyway, so this may have little 'real world' impact/benefit.
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/pull/1858#discussion_r673613538_